engine/actions_test.go

```diff
@@ -1468,6 +1467,7 @@ func TestTopupActionLoaded(t *testing.T) {
 	}
 }
 
+/* Disabled tests with cdrLogAction because it need a rpc connection
```

Comment:
please remove completely the commented test, so we don't have useless code there